src/database.ts

```diff
@@ -28,7 +28,7 @@ import * as extend from 'extend';
 import * as r from 'teeny-request';
 import * as streamEvents from 'stream-events';
 import * as through from 'through2';
-import {Operation as GaxOperation} from 'google-gax';
+import {grpc, Operation as GaxOperation} from 'google-gax';
```

Comment:
Seems needs to import `CallOptions` as well to be used with `restore()` `options` later on